src/client/components/terminal/index.ts

```diff
@@ -878,23 +923,35 @@ export class TerminalView extends LitElement {
           () => {},
         )}
         ${when(
-          this.exitCode !== 0 &&
+          features.isOn(FeatureName.Escalate, this.featureState$) &&
+            this.exitCode !== 0 &&
+            !this.escalationUrl &&
             this.platformId &&
-            !this.isDaggerOutput &&
-            features.isOn(FeatureName.Escalate, this.featureState$),
+            !this.isDaggerOutput,
```

Comment:
@pastuxso Should I validate `!this.isDaggerOutput` for the other buttons as well?

Reply:
@peraltafederico If the button implies something from the terminal, yes. But, if the button only performs a mutation without any dependency of the terminal, no.

Reply:
@pastuxso Cool. Is it possible to have `this.platformId` and `this.isDaggerOutput` both set to `true`?

Reply:
@peraltafederico No, but you can validate it running the Dagger example. Let me know if you need help running the Dagger examples.